composer.json

```diff
@@ -14,18 +14,17 @@
     "minimum-stability": "dev",
     "prefer-stable": false,
     "require": {
-        "php": "^5.3.9|^7.0",
+        "php": "^5.5.9|^7.0",
         "symfony-cmf/routing": "~1.2",
-        "symfony/framework-bundle": "~2.3"
+        "symfony/framework-bundle": "~2.8|~3.0"
     },
     "require-dev": {
-        "symfony-cmf/core-bundle": "~1.1",
+        "symfony-cmf/core-bundle": "dev-sf_3@dev",
         "symfony-cmf/testing": "~1.2,>=1.2.7",
         "doctrine/phpcr-odm": "~1.3",
         "matthiasnoback/symfony-dependency-injection-test": "~0.6",
-        "matthiasnoback/symfony-config-test": "0.*",
+        "matthiasnoback/symfony-config-test": "^1.3.1",
         "phpunit/php-code-coverage": "@stable",
-        "sonata-project/doctrine-phpcr-admin-bundle": "~1.1",
```

Comment:
this should not be removed. If we update this to `^1.1|^2.0`, I think we'll have Symfony 3 support.